docs/app/pages/components/sections/panels/item-display-panel/snippets/app.ts

```diff
@@ -126,6 +130,24 @@ export class AppComponent {
         this.updatePanel();
     }
 
+    upArrow(event: KeyboardEvent) {
+        if (this.visible) {
+            event.preventDefault();
+            if (this.previousEnabled) {
```

Comment:
We should probably remove this `if (this.previousEnabled)` from this function and instead in the `previous()` function we should have it check if it can move back or not instead (do the same for the next button too) eg,

```typescript
previous() {
     let id = this.selectedItem && this.selectedItem.id > 0 ? this.selectedItem.id - 1 : 0;
     this.selectedItem = this.items[id - 1];
     this.updatePanel();
}
```